frontend/src/API/DevPortfolioAPI.ts

```diff
@@ -1,8 +1,8 @@
 import APIWrapper from './APIWrapper';
 import { backendURL } from '../environment';
 
-type DevPortfolioResponseObj = {
-  devPortfolio: DevPortfolio;
+type DevPortfolioSubmissionResponseObj = {
+  submission?: DevPortfolioSubmission;
   error?: string;
 };
```

Comment:
I don't think this is needed

Reply:
Originally was trying to use that for error-handling, if there's an error display it, else use the submission. Is there a better way?

Reply:
I could be wrong but I think [this](https://github.com/cornell-dti/idol/blob/main/frontend/src/API/APIWrapper.ts#L47-L57) should already deal with error handling.